pypeit/core/extract.py

```diff
@@ -1068,8 +1068,13 @@ def create_skymask_fwhm(sobjs, thismask, box_pix=None):
         med_fwhm = np.median(all_fwhm)
         # Boxcar radius?
         if box_pix is not None:
-            med_fwhm = max(med_fwhm, box_pix)
-        msgs.info("Masking around the object with {} pixels".format(med_fwhm))
+            # DP: If we want to enforce the boxcar masking this will prevent it if FWHM is larger.
+            # I think here med_fwhm should be = box_pix
```

Comment:
I trust you.

Reply:
I modified the code and docs slightly preserving your functionality. Mainly variable names were changed.